kotsadm/web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -63,10 +62,19 @@ class AppVersionHistory extends Component {
     yamlErrorDetails: [],
     deployView: false,
     selectedSequence: "",
-    releaseWithErr: {}
+    releaseWithErr: {},
+
+    versionHistoryJob: new Repeater(),
+    loadingVersionHistory: true,
+    versionHistory: [],
+    errorTitle: "",
+    errorMsg: "",
+    displayErrorModal: false,
   }
 
   componentDidMount() {
+    this.startVersionHistoryJob();
```

Comment:
just curious, why not just call this immediately?
```
this.state.versionHistoryJob.start(this.fetchKotsDownstreamHistory, 2000);
```

Reply:
also, this should not run when the component mounts, only when `isAwaitingResults` is `true`. and should be stopped when `isAwaitingResults` becomes `false`